manager/integration/tests/test_settings.py

```diff
@@ -39,9 +44,9 @@ def test_setting_toleration():
     14. Attach the volume and validate `data2`.
     15. Generate and write `data3` to the volume.
     """
-    client = get_longhorn_api_client()
+    client = get_longhorn_api_client()  # NOQA
```

Comment:
Why do we need `# NOQA` here and lines below?

Reply:
pytest will complain about redefined client, since at the beginning we imported it from common.